python/cudf/cudf/utils/ioutils.py

```diff
@@ -81,7 +81,7 @@
 See Also
 --------
 cudf.read_csv
-cudf.read_json
+cudf.io.json.read_json
```

Comment:
Question for reviewers: Why are `cudf.read_csv` and `cudf.io.json.read_json` in the "See Also" section of the `read_avro` docs? Should we remove this? I don't see the relationship, but I'm not familiar with the Avro format.

Reply:
Yeah, remove them.

Reply:
Good catch, I don't think there's a good reason to mention CSV and JSON here.
Why are namespaces(?) so different between csv and json?

Reply:
> Why are namespaces(?) so different between csv and json?

It seems to me that `cudf.io.json.read_json` should be named `cudf.read_json` to mirror pandas. https://pandas.pydata.org/docs/reference/api/pandas.read_json.html

I can fix that in a separate PR, if you like. This one has a nice small scope.

Reply:
https://github.com/rapidsai/cudf/pull/10640